### PR TITLE
Closes #68 #97 and #98 - Create Routes.jsx and updated links across all components using…

### LINKED ISSUE #68: make all route string consts (and centralize them)
See the Main2 and App components for reference.

### PR DESCRIPTION
**Pull request by alexaiconejo**
Created the Routes.jsx component, placed it within src/components/Routes.jsx; it could be moved to src/routes, as preferred. Replaced links across all components (Analisis, Conecta, Footer, Main2, Marquee, Navbar, Notas, as well as App and Main), and it's working fine.

The issue still pending is that the links to "analisis" and "main2" currently function as anchors with the ids "analisis" and "Main2" within App.jsx using a scroll link. For now, I replaced them as they were, but it would be necessary to modify them to eliminate the ids and make them work similarly within any component, not just within App. The same issue exists with issues 97 and 98; they don't work because they should be scroll links within the same component. When redirecting to the same route, it doesn't return to the beginning of the App.jsx component.

### CODE REVIEW

**Comment by alexaiconejo** [comment]
Updated the ids for "analisis" and "Main2" to "analisisAnchor" and "main2Anchor" respectively in App.jsx, 
   and incorporated them as routes within Routes.jsx. Additionally, introduced the "homeAnchor" route within App.jsx,
   redirecting to the first component for scroll linking with that id. That resolved issues #97 and #98.

**Comment by alexaiconejo** [comment]
Renamed Routes.jsx to index.js and moved it to src/routes/index.js.
Replaced all imports accordingly.
Exported an object named Navlinks containing all the previous constants.
Replaced the routes from SomeRoute to Navlinks.SomeRoute.

**Review by xaiki — CHANGES_REQUESTED** [review]
we're already in the NavLinks object,

AnchorRoute makes no sense
EmailRoute either

probably better to remove all "Route" suffixes.

File: src/App.jsx [hunk 1]
```diff
@@ -155,7 +155,7 @@ function App() {
   };
 
   return (
-    <div className={styles.App}>
+    <div id="homeAnchor" className={styles.App}>
```
Comment (xaiki) [inline src/App.jsx:158]: here you want to use the string from NavLinks
DRY: Donot Repeat Yourself

File: src/components/Analisis.jsx [hunk 2]
```diff
@@ -1,11 +1,12 @@
 import styles from "./Analisis.module.css";
 import { Link as ScrollLink } from "react-scroll";
 import PropTypes from "prop-types";
+import Navlinks from "../routes/index.js";
 
 export default function Analisis({ min, max, total, componentes }) {
   return (
     /* TODO: extract this ID, "analisis", which is also used in Main2 and App, to a constant */
-    <div id="analisis" className={styles.analisis}>
+    <div id="analisisAnchor" className={styles.analisis}>
```
Comment (xaiki) [inline src/components/Analisis.jsx:9]: ditto

File: src/components/Main2.jsx [hunk 3]
```diff
@@ -3,27 +3,28 @@ import { Link as ScrollLink } from "react-scroll";
 import { Link } from "react-router-dom";
 import { Carousel } from "react-bootstrap";
 import { Details } from "./Conecta.jsx";
+import Navlinks from "../routes/index.js";
 
 export default function Main2() {
   return (
     // TODO move this ID, Main2, to its own constant
-    <div id="Main2" className={styles.Main2}>
+    <div id="main2Anchor" className={styles.Main2}>
```
Comment (xaiki) [inline src/components/Main2.jsx:11]: ditto

File: src/components/Navbar.jsx [hunk 4]
```diff
@@ -5,14 +5,15 @@ import { Link as ScrollLink } from "react-scroll";
...
+import Navlinks from "../routes/index.js";
 
 export default function NavbarBootstrap() {
   return (
     <div className={styles.heading}>
       <div>
         {/* TODO: move these hrefs to constants. Check whether the nested Brand and Link work as intended. */}
         <Navbar.Brand href="#home">
-          <Nav.Link href="/">
+          <Nav.Link as={Link} to={Navlinks.home}>
```
Comment (xaiki) [inline src/components/Navbar.jsx:16]: why did you add as={Link} ?

File: src/main.jsx [hunk 5]
```diff
@@ -29,11 +30,11 @@ const router = createHashRouter([
...
     element: <Root />,
     children: [
-      { path: "/", element: <App />, loader },
-      { path: "/conecta", element: <Conecta /> },
-      { path: "/reporta", element: <Reporta /> },
-      { path: "/notas", element: <Notas /> },
-      { path: "/main2", element: <Main2 /> },
+      { path: Navlinks.home, element: <App />, loader },
+      { path: Navlinks.conecta, element: <Conecta /> },
+      { path: Navlinks.reporta, element: <Reporta /> },
```
Comment (xaiki) [inline src/main.jsx:35]: this will break
Navlinks.reporta links to google docs, not to /reporta

File: src/main.jsx [hunk 6]
```diff
@@ -29,11 +30,11 @@ const router = createHashRouter([
...
+      { path: Navlinks.notas, element: <Notas /> },
+      { path: Navlinks.main2, element: <Main2 /> },
```
Comment (xaiki) [inline src/main.jsx:37]: this will break,
Navlinks.main2 is main2Anchor not /main2

File: src/main.jsx [hunk 7]
```diff
@@ -29,11 +30,11 @@ const router = createHashRouter([
     path: "/",
```
Comment (xaiki) [inline src/main.jsx:30]: this whole object should probably moved to routes/
but that can be done later.

**Review by xaiki — CHANGES_REQUESTED** [review]
we need to use the consts everywhere for them to make sense.
please check that the router change are actually equivalent.

**Review by xaiki — APPROVED** [review]
lgtm,

this is a fairly large PR
congrats on getting it through.

**Pull request merged**